var/spack/repos/builtin/packages/py-entity-management/package.py

```diff
@@ -12,8 +12,7 @@ class PyEntityManagement(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/nse/entity-management'
     git      = 'ssh://bbpcode.epfl.ch/nse/entity-management'
 
-    version('1.0.5', tag='entity-management-v1.0.5', preferred=True)
-    version('0.1.12', tag='entity-management-v0.1.12')
+    version('1.0.9.dev1', tag='entity-management-v1.0.9.dev1')
```

Comment:
I wonder about this version naming from module perspectives. Any possibility to have "proper" version sooner?

Reply:
@pramodk if you mean `dev` suffix then I remove it before merging and after I test the module jenkins builds on bb5. Unfortunately jenkins job didn't run yet, so I'm waiting for the magic to happen... :)

Reply:
As in the tag `…dev1` becomes the stable tag in your `git`? If you have your own CI for tests, this shouldn't really be needed, as the pull request modules are never exposed to end-users.

Reply:
Yes, `dev` tags are stable. And I prefer to run integration tests with the newly build modules to make sure they actually work in the pipelines. If I see fixes are needed I bump the version in the same PR(as it is not possible to rebuild the same version) and retest. When things work I make a release, drop `dev` suffix from the module name and merge. I think that captures the workflow I established empirically of how to work with spack in the BBP context. Suggestions are more than welcome :)

Reply:
Sounds fine to me, actually. The one suggestion I'd have is to maybe just push changes (not force-push)? The latter seems to mess with the notifications/interactions on GH a little too much, IMO.

There should be some tighter module integration possible with team CI workflows, but I haven't put any work towards coming up with a good setup and documenting such a thing.

Reply:
(the thought here being that prior to posting a PR here you'll already know that your workflows work with resulting spack modules)

Reply:
Thanks  @matz-e for the comments, note taken on no force-push :)
Prior of making a PR I do build/unit test spack package locally and then I rely on jenkins to build the env module which I can use in the pipeline integration test with custom MODULEPATH. Probably I could build modules myself somehow... looking into that at the moment.

Reply:
Ok, I found the modules ;) So I should be able to do all the different types of test before making a PR.